llvm/include/llvm/CodeGen/IntrinsicLowering.h

```diff
@@ -15,17 +15,29 @@
 #ifndef LLVM_CODEGEN_INTRINSICLOWERING_H
 #define LLVM_CODEGEN_INTRINSICLOWERING_H
 
+#include "llvm/ADT/ArrayRef.h"
+
 namespace llvm {
 class CallInst;
 class DataLayout;
+class Type;
+class Value;
 
 class IntrinsicLowering {
   const DataLayout &DL;
+  bool LowerExternalFunctionCalls;
```

Comment:
This should be called something like `AllowLibraryFunctionCalls` as these function calls are created, not lowered.
Also, the new argument warrants a comment explaining what it does.